Comanche055/REENTRY_CONTROL.agc

```diff
@@ -54,8 +54,8 @@ STARTENT	EXIT			# MM = 63
 		INHINT
 		MASK	CM/FLAGS
 					# ENTRYDSP = 92D B13
-					# GONEPAST=95D B10	RELVELSW=96D B9
-					# EGSW = 97D B8
+					# GONEPAST=95D B10.	RELVELSW=96D B9
+					# EGSW = 97D B8         NOSWITCH = 98D B7
```

Comment:
This line requires tabs instead of spaces to match line 57.

Reply:
Changed to tabs.